### ISSUE TITLE
vagrant up issue with vagrant-triggers plugin

### ISSUE THREAD

**Issue by andreagrillo** [Varying-Vagrant-Vagrants/VVV#1477]
I have an issue when I run vagrant up command. I see this issue on consolle:
https://www.screencast.com/t/IaGcHfuf9d6
but the vagrant-triggers plugin is required one.

Another strange issue is that I haven't any VM here:
https://www.screencast.com/t/yNNSml2b25n

Any idea ?

## Your Environment

* VVV version: 2.1.0 
* VVV Git Branch: master 
* Vagrant version: 
* VM Provider name: virtualbox 
* VM Provider version: 5.2.10
* Operating System and version: windows 10 - latest release

**Comment by tomjn** [comment]
Hello!

This is because you upgraded to Vagrant 2.1, we can fix it for 2.1 but that
would break it for everybody else and require a new release, there's a big
warning about it as soon as you run the command

Additionally if it fails there why would it continue and create a VM?

At the moment the only options are to either downgrade or wait for somebody
to make the changes in the develop branch

As an aside, you can upload images inline on Github, you don't have to
upload elsewhere. My response was almost that I don't have the bandwidth in
my phone on a train to load a video screen cast, copy pasting the output
helps

**Comment by andreagrillo** [comment]
Hi there, thanks for your reply. 

How -can I downgrade to previous version ? 
However, this is the two screenshots

![immagine](https://user-images.githubusercontent.com/14902963/39663131-c4e8dab6-506d-11e8-8f35-e944a764f99e.png)

![immagine](https://user-images.githubusercontent.com/14902963/39663135-cfaeb9a2-506d-11e8-952c-49aa9c9c8610.png)

Let me know and thanks for your help!
AG

**Comment by andreagrillo** [comment]
This is the output:

`
WARNING: Vagrant has detected the `vagrant-triggers` plugin. This plugin conflicts
with the internal triggers implementation. Please uninstall the `vagrant-triggers`
plugin and run the command again if you wish to use the core trigger feature. To
uninstall the plugin, run the command shown below:

  vagrant plugin uninstall vagrant-triggers

Note that the community plugin `vagrant-triggers` and the core trigger feature
in Vagrant do not have compatible syntax.

To disable this warning, set the environment variable `VAGRANT_USE_VAGRANT_TRIGGERS`.


  __     ___     ___     __  ____
  \ \   / \ \   / \ \   / / |___ \
   \ \ / / \ \ / / \ \ / /    __) |
    \ V /   \ V /   \ V /    / __/
     \_/     \_/     \_/    |_____|

  Varying Vagrant Vagrants 2.0.0

  Docs:       https://varyingvagrantvagrants.org/
  Contribute: https://github.com/varying-vagrant-vagrants/vvv



Bringing machine 'default' up with 'virtualbox' provider...
C:/HashiCorp/Vagrant/embedded/gems/2.1.0/gems/vagrant-2.1.0/lib/vagrant/plugin/v2/config.rb:73:in `method_missing': undefined method `before_triggers' for #<VagrantPlugins::Triggers::Config::Trigger:0x0000000004509ef8> (NoMethodError)
        from C:/HashiCorp/Vagrant/embedded/gems/2.1.0/gems/vagrant-2.1.0/lib/vagrant/plugin/v2/trigger.rb:40:in `fire_triggers'
        from C:/HashiCorp/Vagrant/embedded/gems/2.1.0/gems/vagrant-2.1.0/lib/vagrant/machine.rb:164:in `action'
        from C:/HashiCorp/Vagrant/embedded/gems/2.1.0/gems/vagrant-2.1.0/lib/vagrant/batch_action.rb:82:in `block (2 levels) in run'
`

**Comment by andreagrillo** [comment]
I've found the final solution for that:

1. Remove the current Vagrant 2.1.0 and reboot the system

2. From here https://releases.hashicorp.com/vagrant/2.0.4/ download the previous version of Vagrant for your OS

3. Install it and reboot the system

4. Add again the vagrant plugin with this command:

`vagrant plugin install vagrant-hostsupdater vagrant-triggers vagrant-vbguest`

(Look here for more information: https://varyingvagrantvagrants.org/docs/en-US/installation/software-requirements/)

and reboot the system

5. IMPORTANT: open a git bash like administrator and follow the vvv installation "how to" at this link: https://varyingvagrantvagrants.org/docs/en-US/installation/

![image](https://user-images.githubusercontent.com/14902963/39663878-0928ff50-507b-11e8-8b63-3fd1a922f5fc.png)

![image](https://user-images.githubusercontent.com/14902963/39663888-35ff66d6-507b-11e8-900c-7c0f10e7bbdc.png)

After `vagrant up --provision` complete all operation, vagrant start to works again.

**Comment by mourato** [comment]
Hi, guys,

I'm with the same problem as well.

The log is:

```
Bringing machine 'default' up with 'virtualbox' provider...
/opt/vagrant/embedded/gems/2.1.0/gems/vagrant-2.1.0/lib/vagrant/plugin/v2/config.rb:73:in `method_missing': undefined method `before_triggers' for #<VagrantPlugins::Triggers::Config::Trigger:0x00000000026acb90> (NoMethodError)
        from /opt/vagrant/embedded/gems/2.1.0/gems/vagrant-2.1.0/lib/vagrant/plugin/v2/trigger.rb:40:in `fire_triggers'
        from /opt/vagrant/embedded/gems/2.1.0/gems/vagrant-2.1.0/lib/vagrant/machine.rb:164:in `action'
        from /opt/vagrant/embedded/gems/2.1.0/gems/vagrant-2.1.0/lib/vagrant/batch_action.rb:82:in `block (2 levels) in run'
```

### My Environment
VVV version: 2.1.0
VVV Git Branch: master
Vagrant version: 2.1
VM Provider name: virtualbox
VM Provider version: 5.2.10
Operating System and version: Ubuntu 16.40

**Comment by tomjn** [comment]
**VVV 2.1 and Vagrant 2.1 will never work together**

Downgrade to 2.0.2 or wait for 2.2

**Comment by andreagrillo** [comment]
@mourato 

I have fixed the issue with the Vagrant Version 2.0.4, look my previous reply. I hope this works for you, too. 
AG

**Comment by mourato** [comment]
Got it, guys. For now, I will downgrade Vagrant and wait for 2.2.

Thanks :-)
